chat-ui/src/DevCenterChatbot.tsx

```diff
@@ -164,10 +180,17 @@ type InnerChatbotProps = {
 const WELCOME_MESSAGE =
   "Welcome to MongoDB AI Assistant. What can I help you with?";
 
-export function InnerChatbot({
+const SUGGESTED_PROMPTS = [
+  "How do you deploy a free cluster in Atlas?",
+  "How do you import or migrate data into MongoDB Atlas?",
+  "How do I get started with MongoDB?",
+  "Why should I use Atlas Search?",
+];
```

Comment:
For context, we now take these in as a prop on the docs chatbot. I think it's fine to hard code these for now but in our unified interface we'll probably want to use the props approach for both.